### PR TITLE
refactor(core): patch file for removal of id from schema

### PR DESCRIPTION
**Pull request by prajjwalkumar17**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

This PR includes a patch file for removal of id from tables.

## Tables modified for API-V2:
**blocklist**
```
unique_key (merchant_id, fingerprint_id)
primary_key (merchant_id, fingerprint_id)
```

**payment_methods**
```
no unique_key 
primary_key (payment_method_id)
```

**refund**
```
unique_key (refund_id, merchant_id) 
primary_key (merchant_id, refund_id)
```

**user**
```
unique_key (user_id)
primary key (user_id)
```

**user_roles** 
```
unique_key (user_id, merchant_id)
primary_key (user_id, merchant_id)
```

**roles**
```
unique_key  (role_id)
primary_key (role_id)

```
### Additional Changes

- [ ] This PR modifies the API contract
- [x] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context

required for v2

## How did you test it?

The code compilation works fine, and moreover it is id exclusion so we can probably skip testing.

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: crates/diesel_models/remove_id.patch [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+     roles (role_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         role_name -> Varchar,
+         #[max_length = 64]
+@@ -1276,7 +1263,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     users (user_id) {
+-        id -> Int4,
```
Comment (Narayanbhat166) [inline crates/diesel_models/remove_id.patch:105]: @ThisIsMani This id is not used anywhere right?

**Review by Narayanbhat166 — DISMISSED** [review]

File: crates/diesel_models/remove_id.patch (hunk 1 above)
Comment (ThisIsMani) [reply crates/diesel_models/remove_id.patch:105]: No, we are not.

**Review by ThisIsMani — COMMENTED** [review]
Dashboard specific changes looks fine.

**Review by SanchithHegde — DISMISSED** [review]

**Comment by prajjwalkumar17** [comment]
> Dashboard specific changes looks fine.

Sure, can you review/approve this?

**Review by Narayanbhat166 — APPROVED** [review]

**Review by ThisIsMani — DISMISSED** [review]

**Review by Narayanbhat166 — DISMISSED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Pull request merged**